src/Xamarin.Android.Build.Tasks/Tasks/Aot.cs

```diff
@@ -405,9 +405,9 @@ IEnumerable<Config> GetAotConfigs ()
 					aotOptions.Add ("asmwriter");
 					aotOptions.Add ($"mtriple={mtriple}");
 					aotOptions.Add ($"tool-prefix={toolPrefix}");
-					aotOptions.Add ($"ld-flags={ldFlags}");
 					aotOptions.Add ($"llvm-path={sdkBinDirectory}");
 					aotOptions.Add ($"temp-path={tempDir}");
+					aotOptions.Add ($"ld-flags={ldFlags}");
```

Comment:
@radekdoulik why did we need to change the order? Is the order important?

Reply:
I am just experimenting, curious whether it would help with `llvm-path`. Next I am going to try escape the quotes with '^' char instead of '\\'